docs/source/a_quick_tour.mdx

```diff
@@ -200,6 +199,47 @@ The `combine` function accepts both the list of names of the metrics as well as
 }
 ```
 
+## Running evaluation on a suite of tasks
+
+It can be useful to evaluate models on a variety of different tasks to understand their downstream performance. The [EvaluationSuite](evaluation_suite) enables evaluation of models on a collection of tasks. Tasks can be constructed as ([evaluator](base_evaluator), dataset, metric) tuples and passed to an [EvaluationSuite](evaluation_suite) stored on the Hugging Face Hub as a Space. See the [evaluator documentation](base_evaluator) for a list of currently supported tasks.
```

Comment:
The scripts can be on the hugging face hub or defined locally, right?

Reply:
Yup, good catch.